src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -7,17 +8,33 @@ public class LayoutView : MauiView
 	{
 		bool _userInteractionEnabled;
 
+		public LayoutView()
+		{
+			Descendants = new List<UIView>();
+		}
+		
+		internal List<UIView>? Descendants { get; set; }
+
+		protected override void Dispose(bool disposing)
+		{
+			base.Dispose(disposing);
+
+			Descendants = null;
+		}
+
 		public override void SubviewAdded(UIView uiview)
 		{
 			InvalidateConstraintsCache();
 			base.SubviewAdded(uiview);
+			Descendants?.Add(uiview);
```

Comment:
How is this List different then just checking "this.SubViews" ?

Reply:
No differences, updated.